m2cgen/interpreters/c/code_generator.py

```diff
@@ -4,10 +4,20 @@
 from m2cgen.interpreters.code_generator import CodeTemplate as CT
 
 
+ARRAY_ASSIGNMENT_FUNC = """
+void assignArray(double *output, double input[], int size) {
```

Comment:
Snake case is a lot more common in C. Let's make it `assign_array`.

Reply:
Done